Tasks/Popov/Task1/Homework1.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+# Formula for calculating the amount without including capitalization
+
+# S = P + (P * I * t / K) / 100
+
+# Name
+  
+'''S - the final amount received upon completion of the deposit;
+   P - the amount initially deposited;
+   I - the size of the % rate (per year);
+   t - number of days of accrual%;
+   K - number of days per calendar year.'''
+
+P = 20000
+I = 15
+t = 1825
+K = 365
```

Comment:
these values must be entered by a user, -10 points for missing part of a task

also, it's a money-related operations, so it must be dealt in decimal, -1 point